test.py

```diff
@@ -263,6 +263,10 @@ def __init__(self, path: str, cfg: dict, options: argparse.Namespace, mode: str)
         self.all_can_run_compaction_groups_except = cfg.get("all_can_run_compaction_groups_except")
 
     async def create_test(self, shortname, suite, args):
+        exe = os.path.join("build", suite.mode, "test", suite.name, shortname)
+        if not os.path.exists(exe):
```

Comment:
Perhaps add os.access(fpath, os.X_OK) to check if the file is runnable on the same token.

Reply:
Will do in a follow-up